python-frontend/src/main/java/org/sonar/python/semantic/v2/ProjectLevelTypeTable.java

```diff
@@ -61,4 +62,30 @@ public ModuleType getModule(List<String> moduleNameParts) {
     }
     return parent;
   }
+
+  public PythonType getType(String typeFqn) {
```

Comment:
I think the approach makes sense, although I think we'll need to be careful about using this notion of a fully qualified name since we haven't finished defining it yet.
In this form, it probably only make sense while we still rely on the previous `ProjectLevelSymbolTable` conversion.